scripts/release/README.md

```diff
@@ -19,66 +19,71 @@ The high level process of creating releases is [documented below](#process). Ind
 
-<sup>1: You can omit the `build` param if you just want to release the latest commit as a canary.</sup>
+<sup>1: This is the most awkward part of cutting a release right now. We have plans to improve it.</sup><br/>
+<sup>2: You can omit the `build` param if you just want to release the latest commit as to "next".</sup>
 
-## Publishing an Experimental Canary
+## Publishing an Experimental Release
 
-Experimental canaries are special releases with additional features turned on.
+Experimental releases are special because they have additional features turned on.
 
-The steps for publishing an experimental canary are almost the same as for publishing a normal canary, except in step 3 you should choose the `build_experimental` job instead of `build`. (I know this is awkward; we have plans to make it less so. Ideally these canaries would get published by a cron job.)
+The steps for publishing an experimental release are almost the same as for publishing a "next" release, except in step 3 you should choose the `build_experimental` job instead of `build` <sup>1</sup> For example, the experimental build ID for commit [241c4467](https://github.com/facebook/react/commit/241c4467eef7c2a8858c96d5dfe4e8ef84c47bad) is [**80590**](https://circleci.com/gh/facebook/react/80590):
 
-When publishing an experimental canary, use the `experimental` tag:
+When publishing an experimental release, use the `experimental` tag:
 
 ```sh
 scripts/release/publish.js --tags experimental
 ```
 
 If the OTP code expires while publishing, re-run this command and answer "y" to the questions about whether it was expected for already published packages.
 
+<sup>1: We have plans to make this less awkward. Ideally these releases will be published by a cron job.</sup>
+
 ## Publishing a Stable Release
 
-Stable releases should always be created from a previously-released canary. This encourages better testing of the actual release artifacts and reduces the chance of unintended changes accidentally being included in a stable release.
+Stable releases should always be created from the "next" channel. This encourages better testing of the actual release artifacts and reduces the chance of unintended changes accidentally being included in a stable release.
 
-To prepare a stable release, choose a canary version and  run the [`prepare-stable`](#prepare-stable) script <sup>1</sup>:
+To prepare a stable release, choose a "next" version and run the [`prepare-stable`](#prepare-stable) script <sup>1</sup>:
 
 ```sh
-scripts/release/prepare-stable.js --version=0.0.0-5bf84d292
+scripts/release/prepare-stable.js --version=0.0.0-241c4467e
 ```
 
 This script will prompt you to select stable version numbers for each of the packages. It will update the package JSON versions (and dependencies) based on the numbers you select.
 
 Once this step is complete, you're ready to publish the release:
 
 ```sh
-scripts/release/publish.js --tags next latest
+scripts/release/publish.js --tags latest
```

Comment:
Note this is arguably a significant change. If "next" releases are identified as always being 0.0.0-<hash>, and "latest" releases are always semver numbers, it doesn't make sense for us to update `next` to match `latest` on a stable release. I think that's fine though.